components/style/selector_matching.rs

```diff
@@ -380,15 +380,15 @@ impl Stylist {
                             return true
                         }
                     }
-                    mq_eval_changed(rules, before, after)
+                    mq_eval_changed(&rules, before, after)
```

Comment:
Your patch compiles with and without this change. I guess it's not needed, but can you explain why that is?